MAUI/DigitalGauge/Customize-character-segments.md

```diff
@@ -0,0 +1,268 @@
+
+    <gauge:SfDigitalGauge Text="SYNCFUSION" StrokeWidth="4" 
+                          HeightRequest="120"  WidthRequest="330" 
+                          BackgroundColor="Black"
+                          DisabledSegmentAlpha="0.1" DisabledSegmentStroke="#146CED"
+                          CharacterHeight="117" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="SevenSegment" 
+                          CharacterStroke="#146CED"/>
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 120;
+
+            digital.WidthRequest = 330;
+
+            digital.Text = "SYNCFUSION";
+
+            digital.CharacterHeight = 117;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.StrokeWidth = 4;
+
+            digital.CharacterType = DigitalGaugeCharacterType.SevenSegment;
+
+            digital.DisabledSegmentAlpha = 0.1;
+
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Setting character spacing
+
+The values of digital characters are spaced by altering the space of digital characters. It is achieved by setting the `CharacterSpacing` property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+     <gauge:SfDigitalGauge Text="01-01-24" StrokeWidth="3" 
+                          HeightRequest="100"  WidthRequest="360" CharacterSpacing="10"
+                          DisabledSegmentAlpha="0.1" DisabledSegmentStroke="#146CED"
+                          CharacterHeight="90" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
+                          CharacterType="EightCrossEightDotMatrix"                
+                          CharacterStroke="#146CED"/>
+{% endhighlight %}
+
+{% highlight c# %}
+
+            SfDigitalGauge digital = new SfDigitalGauge();
+
+            digital.HeightRequest = 100;
+
+            digital.WidthRequest = 360;
+
+            digital.Time = "01-01-18";
+
+            digital.CharacterHeight = 90;
+
+            digital.CharacterWidth = 25;
+
+            digital.HorizontalOptions = LayoutOptions.Center;
+
+            digital.VerticalOptions = LayoutOptions.Center;
+
+            digital.StrokeWidth = 3;
+
+            digital.CharacterSpacing = 10;
+
+            digital.CharacterType = DigitalGaugeCharacterType.EightCrossEightDotMatrix;
+
+            digital.DisabledSegmentAlpha = 0.1;
+
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Customize character segment stroke
```

Comment:
Customize character segment stroke and stroke width

add it in here

The width of the digital character value can be customized using the StrokeWidth property.